packages/cape-verde/cape-verde.svg

```diff
@@ -0,0 +1,96 @@
+<svg
+		xmlns="http://www.w3.org/2000/svg"
+		viewBox="0 0 860 793"
+		width="859.93915"
+		height="792.58252"
+		aria-label="Map of Cape Verde"
+>
+	<path
+			d="m 216.68917,759.77285 0.71,1 -1.06,0.82 0.27,0.46 1.06,-0.46 0.44,0.73 1.42,0.18 0.44,0.55 -0.97,0.46 -0.27,0.82 -0.97,-0.18 -0.18,0.73 0.89,0.27 -0.35,0.64 0.18,2.73 0.89,0.09 0.18,1.18 0.71,0.91 -0.09,0.82 1.33,1 0.44,1.27 -0.09,1.37 -0.88,1.64 0.62,0.73 0.35,0.36 0.8,0.64 -0.35,0.82 -1.06,0 -0.27,1 0.53,1 0,0.55 -0.44,0.82 -1.42,0.46 0.44,1 -0.71,2 0.27,0.46 -1.24,0.18 -0.62,1 -0.62,-0.27 -1.24,0.82 -0.44,0.55 -0.44,0.36 -0.53,0 -0.44,0.82 -0.88,-0.18 -0.71,0.73 -0.35,0.55 -0.97,0.09 0,0.73 -0.62,0.27 -1.24,-1 0.62,-0.64 -0.18,-1.27 -2.04,0.36 -0.27,0.64 -0.53,-0.55 -0.62,0.64 -0.09,-0.91 -1.24,0.09 -0.09,-0.73 -0.62,-0.09 -1.24,-0.46 -0.09,-1.18 -1.42,-1.18 -0.44,-1.64 -0.71,-0.46 -0.97,-1.82 -0.8,0.18 -0.62,1.37 -2.04,-1.73 0,-2.28 0.71,0.18 0.35,-2.82 -1.33,-1 0.18,-0.82 0.88,-0.27 -1.24,-1.18 0.53,-0.73 -0.27,-1.27 0.44,-0.18 -0.44,-0.82 0.8,0.27 0.71,-0.73 0.44,0.82 0.97,-0.55 1.06,0.55 1.24,-0.82 0.18,-1.55 -0.97,-0.09 -0.35,-0.73 0.35,-1 -0.27,-1.09 0.8,0.36 0.27,-0.91 1.33,-1.09 -0.09,-1.55 0.44,0.09 -0.09,0.64 0.53,0.09 0,0.36 0.88,-0.18 0.27,0.55 0.8,-1 0.71,0.64 -0.09,0.55 1.06,0.36 0.53,-0.55 -0.18,-0.36 1.24,-1 1.68,-0.27 0.44,-0.45 2.66,-0.09 0.62,-0.46 -0.09,-1.46 1.86,-0.09 0.62,1 0.7,-1.21 z m 7.97,-23.49 0.53,1.55 -1.06,-0.27 -0.71,0.64 -0.53,-0.27 0.18,-1.09 1.37,0.3 -0.22,-0.48 0.44,-0.38 z m 2.83,-0.09 0.53,0.46 0,0.91 -0.97,-0.27 0,-1.09 0.44,0 z m -10.97,-1.82 1.15,0.82 0.09,1 -0.53,1 -0.71,0 0.27,0.73 -0.71,0.73 0.44,0.36 -1.42,1.28 -2.04,0.36 -1.33,-1.37 0.09,-1.18 0.8,0.55 -0.09,-1.09 0.89,-0.73 -0.35,-0.64 0.88,-0.09 -0.09,-0.55 1.15,-1.09 0.53,0.18 0,0.91 0.35,0 0.18,-1 0.45,-0.18 z m 15.4,-2.1 1.15,1.91 0.71,0.09 -0.18,1.64 -1.15,-0.37 0.44,0.82 -0.89,0.46 -0.27,0.73 -0.35,-0.27 -0.35,0.45 -0.3,-0.18 -0.14,1.1 -0.62,0.73 -1.06,-0.73 0,-0.46 -0.97,0.55 0.89,-1.55 0.88,0.09 0.18,-0.64 1.5,-0.73 0.09,-1.09 0.35,-0.09 -0.35,-0.27 0.35,-1.37 -0.44,-0.64 0.53,-0.18 z"
+			name="Brava"
+			id="br"/>
```

Comment:
Are these `id`s common short names?

Reply:
They are not common short names, they are acronyms. When used in the context of administrative divisions of Cape Verde, they clearly identify a division.